### PR TITLE
⭐️ Add GCP monitoring alert policies

### PR DESCRIPTION
**Pull request by imilchev**
```coffee
./cnquery run gcp -c "gcp.project.monitoring.alertPolicies{*}"                                                                                                                                                               
→ discover related assets for 1 asset(s)
→ resolved assets resolved-assets=1
gcp.project.monitoring.alertPolicies: [
  0: {
    updatedBy: "manuel@mondoo.com"
    validity: null
    conditions: [
      0: {
        absent: null
        displayName: "New condition"
        matchedLog: null
        monitoringQueryLanguage: null
        name: "projects/manuel-development-2/alertPolicies/13858885952422700290/conditions/13858885952422701115"
        threshold: {
          comparison: "COMPARISON_GT"
          denominatorFilter: ""
          duration: 0000-01-01 00:17:30 +0017 LMT
          evaluationMissingData: "EVALUATION_MISSING_DATA_UNSPECIFIED"
          filter: "metric.type=\"logging.googleapis.com/user/log-metric-filter-and-alerts-exist-for-project-ownership-assignments-changes\""
          thresholdValue: 1.000000
        }
      }
    ]
    displayName: "alert-ensure-log-metric-filter-and-alerts-exist-for-project-ownership-assignments-changes"
    projectId: "manuel-development-2"
    labels: {}
    combiner: "OR"
    documentation: {
      content: "gcp--ensure-log-metric-filter-and-alerts-exist-for-project-ownership-assignments-changes"
      mimeType: "text/markdown"
    }
    name: "projects/manuel-development-2/alertPolicies/13858885952422700290"
    notificationChannelUrls: []
    updated: 2023-01-18 16:22:51.668249093 +0000 UTC
    enabled: true
    created: 2023-01-18 16:22:51.668249093 +0000 UTC
    alertStrategy: {
      autoClose: 0000-01-08 00:17:30 +0017 LMT
      notificationRateLimit: null
    }
    createdBy: "manuel@mondoo.com"
  }
  1: {
    updatedBy: "manuel@mondoo.com"
    validity: null
    conditions: [
      0: {
        absent: null
        displayName: "New condition"
        matchedLog: null
        monitoringQueryLanguage: null
        name: "projects/manuel-development-2/alertPolicies/14781749476325084227/conditions/14781749476325084442"
        threshold: {
          comparison: "COMPARISON_GT"
          denominatorFilter: ""
          duration: 0000-01-01 00:17:30 +0017 LMT
          evaluationMissingData: "EVALUATION_MISSING_DATA_UNSPECIFIED"
          filter: "metric.type=\"logging.googleapis.com/user/major-foo-1\""
          thresholdValue: 1.000000
        }
      }
    ]
    displayName: "alert-foo-bar-test-1"
    projectId: "manuel-development-2"
    labels: {}
    combiner: "OR"
    documentation: null
    name: "projects/manuel-development-2/alertPolicies/14781749476325084227"
    notificationChannelUrls: [
      0: "projects/manuel-development-2/notificationChannels/9884640812742514901"
    ]
    updated: 2023-01-18 18:40:38.212119022 +0000 UTC
    enabled: true
    created: 2023-01-18 18:40:38.212119022 +0000 UTC
    alertStrategy: {
      autoClose: 0000-01-08 00:17:30 +0017 LMT
      notificationRateLimit: null
    }
    createdBy: "manuel@mondoo.com"
  }
]
```

```coffee
./cnquery run gcp -c "gcp.project.logging.metrics{alertPolicies}"                                                                                                                                                               
→ discover related assets for 1 asset(s)
→ resolved assets resolved-assets=1
gcp.project.logging.metrics: [
  0: {
    alertPolicies: [
      0: gcp.project.monitoringService.alertPolicy id = projects/manuel-development-2/alertPolicies/13858885952422700290
    ]
  }
  1: {
    alertPolicies: [
      0: gcp.project.monitoringService.alertPolicy id = projects/manuel-development-2/alertPolicies/14781749476325084227
    ]
  }
]
```

### CODE REVIEW

**Review by misterpantz — COMMENTED** [review]
Really good docs, @imilchev!

File: resources/packs/gcp/monitoring.go [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+	}
+
+	projectId := provider.ResourceID()
+	(*args)["projectId"] = projectId
+
+	return args, nil, nil
+}
+
+func (g *mqlGcpProject) GetMonitoring() (interface{}, error) {
+	projectId, err := g.Id()
```
Comment (czunker) [inline resources/packs/gcp/monitoring.go:42]: Is this correct?

L18 uses another function to get the projectID:
`g.Id()` vs. `g.ProjectId()`

Comment (imilchev) [reply resources/packs/gcp/monitoring.go:42]: Those are 2 different resources. On L18 we are working with `mqlGcpProjectMonitoringService` that has a property `projectId`. Here we are working with a `mqlGcpProject` that has `id`.

**Review by czunker — APPROVED** [review]
LGTM. Thanks for the new resources

**Pull request merged**